CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0-rc4 (2020-12-02)
+
+This release candidate contains bugfixes since the previous release
```

Comment:
I would expand (a bit dry right now). Maybe say something about improvements to COPY, support for gapfill, and improvements to chunk compression on distributed hypertables?

Reply:
Rewrote it. I'll add info about COPY when it is merged.